gemelli/base.py

```diff
@@ -23,6 +23,10 @@ def fit(self):
     def label(self):
         """ Placeholder for fit this
         should be implemetned by sub-method"""
+    def transform(self):
```

Comment:
I would be __super__ hesitant to simply return `sample_weights` as the `transform` method like this.

Generally the expected behavior of `transform` is "I give you data, and you transform it for me". Your `transform` method is not doing that. Furthermore, `sample_weights` is already accessible as an attribute and you are not doing any extra work on it.

Side note: Is there any way to project new (i.e., not used in  fitting procedure) samples into RPCA/CTF space?

Reply:
These have been removed or altered to match the PCA function in sklearn.